api/src/request.rs

```diff
@@ -0,0 +1,41 @@
+use ethers::types::Bytes;
+use log::{error, info};
+use serde::{Deserialize, Serialize};
+
+#[allow(non_snake_case)]
+#[derive(Serialize, Deserialize)]
+/// Struct to hold the data to be sent to the platform API.
```

Comment:
struct SuccinctProofSerivce {
   rpc_url;
}


impl SuccinctProofService {
   fn new()
}
fn request_call(chainid, to, data, functionid, input)